scripts/config/keyboardArray.js

```diff
@@ -0,0 +1,360 @@
+const keyboardArray = [
+  {
+    code: 'Backquote',
+    ru: 'ё',
+    en: '`',
+    alt: '~',
+  },
+  {
+    code: 'Digit1',
+    ru: '1',
+    en: '1',
+    alt: '!',
+  },
+  {
+    code: 'Digit2',
+    ru: '2',
+    en: '2',
+    alt: '@',
+  },
+  {
+    code: 'Digit3',
+    ru: '3',
+    en: '3',
+    alt: '#',
+  },
+  {
+    code: 'Digit4',
+    ru: '4',
+    en: '4',
+    alt: '$',
+  },
+  {
+    code: 'Digit5',
+    ru: '5',
+    en: '5',
+    alt: '%',
+  },
+  {
+    code: 'Digit6',
+    ru: '6',
+    en: '6',
+    alt: '^',
```

Comment:
all these `alt` keys are wrong for `ru` layout (except exclamation mark, probably). This is a bug, every keyboard should have their alt values.

Reply:
Done. (Do not pay attention, this is just tips for me)